### PR TITLE
Upgrade gRPC to 1.24.1

### PR DESCRIPTION
**Pull request by vmax**
## What is the goal of this PR?

Update gRPC to a more recent version

## What are the changes implemented in this PR?

- Upgrade PyPI dependency on gRPC to 1.24.1
- Bump `@graknlabs_build_tools`, `@graknlabs_protocol ` and `@graknlabs_grakn_core` versions

### CODE REVIEW

File: tests/integration/test_concept.py [hunk 1]
```diff
@@ -163,6 +163,8 @@ def test_set_label(self):
             # set label
             car_type = self.tx.get_schema_concept("car")
             car_type.label("vehicle")
+            # retrieve updated schema concepts
```
Comment (flyingsilverfin) [inline tests/integration/test_concept.py]: why is this needed @vmax?

Comment (vmax) [reply tests/integration/test_concept.py]: @flyingsilverfin
```
q = tx.put_entity_type('q')
tx.get_schema_concept('q') # works
q.label('w')

tx.get_schema_concept('q') # no longer works (expected)
tx.get_schema_concept('w') # ALSO doesn't work
```

however:

```
tx.get_schema_concept('w')  # None
list(tx.query('match $x sub thing; get;')) # list<conceptmap>
tx.get_schema_concept('w') # it is there!
```

which leads me to believe schema concepts might be cached somewhere
